### PR TITLE
SLE 12/15 CCE and CIS numbers for the CIS group job schedulers

### PR DESCRIPTION
**Pull request by rumch-se**
#### Description:

- _Assignment of CCE numbers or update/add of CIS numbers to the SLE 12/15 rules which a part of CIS group 5.1 Configure time-based job schedulers _

#### Rationale:

- Some of SLE 12/15 rules which a part of CIS group 5.1 Configure time-based job schedulers did not have CCE numbers assigned or their CIS numbers were not correct 

#### Review Hints:

- _Review hints here. Replace this text. Don't use the italics format!_

- _Use this optional section to give any relevant information which could help the reviewer to more quickly and assertively understand and test the changes._

- _Good examples are useful commands, if it is better to review all commits together or in a suggested sequence, any relevant discussion in other PRs or issues, etc._

### CODE REVIEW

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

**Comment by rumch-se** [comment]
Hello @marcusburghardt
I have done rebase.
Have a nice day
Rumen

File: shared/references/cce-sle12-avail.txt [hunk 1]
```diff
@@ -1,25 +1,3 @@
...
-CCE-92266-6
-CCE-92267-4
-CCE-92268-2
-CCE-92269-0
-CCE-92270-8
-CCE-92271-6
-CCE-92272-4
-CCE-92273-2
-CCE-92274-0
-CCE-92275-7
```
Comment (marcusburghardt) [inline shared/references/cce-sle12-avail.txt:22]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

File: shared/references/cce-sle15-avail.txt [hunk 2]
```diff
@@ -1,21 +1,3 @@
...
-CCE-91382-2
-CCE-91383-0
-CCE-91384-8
-CCE-91385-5
-CCE-91386-3
-CCE-91387-1
-CCE-91388-9
-CCE-91389-7
-CCE-91390-5
-CCE-91391-3
```
Comment (marcusburghardt) [inline shared/references/cce-sle15-avail.txt:18]: You are probably removing more CCEs than necessary in this PR. Could you check, please?

**Review by marcusburghardt — CHANGES_REQUESTED** [review]

**Comment by rumch-se** [comment]
Hello @marcusburghardt 
I have done rebase and now the numbers used for the rules correspond to the numbers taken from the files with available numbers.
Have a nice day
Rumen

**Review by marcusburghardt — APPROVED** [review]
Thanks

**Pull request merged**